scrape/scrape.go

```diff
@@ -197,6 +199,38 @@ func (sp *scrapePool) stop() {
 	wg.Wait()
 }
 
+func equalRelabelConfigs(as, bs []*config.RelabelConfig) bool {
```

Comment:
This seems a little fragile if we ever add more fields to actions. Could we use deep equal or move this into the relabel package?